### PR TITLE
Add plots of the BH evolution through time

### PR DESCRIPTION
**Pull request by MatthieuSchaller**
Two new plots:
 - BH accretion rate through cosmic time
 - BH mass through cosmic time.

### CODE REVIEW

**Comment by MatthieuSchaller** [comment]
Actually, there might be a funky bug here. If you have a section appearing both in the auto_plotter and the config then you get two sections with the same name (and a broken html anchor).

**Comment by JBorrow** [comment]
Thanks, I've opened up an issue for that. I don't want to address that for this PR though.

File: eagle-xl/scripts/bh_accretion_evolution.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+for index, observation in enumerate(observational_data):
+    obs = load_observation(observation)
+    obs.plot_on_axes(ax)
+
+ax.set_xlabel("Redshift $z$")
+ax.set_ylabel(r"BH Accretion Rate Density [M$_\odot$ yr$^{-1}$ Mpc$^{-3}$]")
+
+redshift_ticks = np.array([0.0, 0.2, 0.5, 1.0, 2.0, 3.0, 5.0, 10.0, 20.0, 50.0, 100.0])
+redshift_labels = [
+    "$0$",
```
Comment (JBorrow) [inline eagle-xl/scripts/bh_accretion_evolution.py:93]: Fwiw these probably don't need to be dollar signed

**Comment by JBorrow** [comment]
Looks good to me. If it makes your life easier remember you can use `{item}.units.latex_repr` to get a pre-written LaTeX representation of a symbolic unit.

**Pull request merged**